test/e2e/knative_eventing_test.go

```diff
@@ -49,10 +49,10 @@ func TestKnativeEventing(t *testing.T) {
 	})
 
 	t.Run("verify correct deployment shape", func(t *testing.T) {
-		// Check the status of scaled deployments in the knative eventing namespace
+		// Check the desired scale of deployments in the knative eventing namespace
 		for _, deployment := range []string{"eventing-controller", "eventing-webhook", "imc-controller", "imc-dispatcher", "mt-broker-controller", "sugar-controller"} {
 			if err := test.CheckDeploymentScale(caCtx, eventingNamespace, deployment, eventingHaReplicas); err != nil {
-				t.Fatalf("Failed to verify default HA settings for `%s`: %v", deployment, err)
+				t.Fatalf("Failed to verify default HA settings for %q: %v", deployment, err)
```

Comment:
In `manifest_test.go` we have also `'%s'` could you fix that too?

Reply:
Yes, sure.